cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1126,6 +1126,22 @@
       },
       "Condition": "CreateEFSSubstack"
     },
+    "CloudWatchLogGroup": {
+      "Type": "AWS::Logs::LogGroup",
+      "Properties": {
+          "LogGroupName": {
+            "Fn::Sub": [
+              "/aws/${cluster_name}",
+              {
+                "cluster_name": {
+                  "Ref": "AWS::StackName"
+                }
+              }
+            ]
+          },
+          "RetentionInDays": 14
```

Comment:
Make into a parameter, or use ssm

Reply:
Agreed that needs to happen eventually. My intent was to merge this version into the feature branch first, and then do that as part of the PR that modifies the user-facing configuration. Thoughts?

Reply:
sounds good